app/src/styles.sass

```diff
@@ -164,17 +163,52 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
   right: 0
   display: flex
   flex-flow: row nowrap
-  width: 80%
+  overflow: hidden
+  width: 80vw
+  height: 150vw
   @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-laptop)
-    width: 50%
+    width: 50vw
+    height: 70vw
   @media screen and (min-width: $breakpoint-laptop)
     display: none
-  .info-panel-icon #help-button
-    margin: 0.7em $small-space
+  .info-panel-content
+    position: absolute
+    z-index: 1
+    right: 0
+    width: 70vw
+    @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-laptop)
+      width: 45vw
+    @media screen and (min-width: $breakpoint-laptop)
+      display: block
   .info-panel-content .information
     display: contents
     p
       margin-top: 1em
+.info-panel.opened  
+  max-width: 80vw
+  min-width:32px
+  @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-laptop)
+    max-width: 50vw
+  @media screen and (min-width: $breakpoint-laptop)
+    display: none
+  .info-panel-icon-open #help-button
+    display: none
+  .info-panel-icon-close
+    position: absolute
+    z-index: 2
+    background-color: #d5b6d0
+    width: 32px
+    height: inherit
```

Comment:
I don't see any reason to have these 5 lines, except the useless lines 175-177. Removing all these 8 lines seems much more simple to me.

Reply:
These 5 lines are used to make sure that the div for the icon are not fulled by the text at the back. If I delete them, the text will come in front of the div of icon and it will looks like this instead. So i think it is useful to keep them
![Capture d’écran 2021-06-18 à 08 33 52](https://user-images.githubusercontent.com/83946943/122517108-eb8e7200-d00f-11eb-8e6f-d84d6bfa7398.png)

Reply:
My suggestion was to remove **8** lines: these lines **and lines 175 to 177**.

Reply:
ah Okay, I see !